enrichment_logbook_uploader/logbook_upload.py

```diff
@@ -109,6 +110,33 @@ def generate_payload(df, logbook_header_id):
         for date in skipped_dates:
             print(date)
 
+    get_logBook_response = requests.post(_ACTIVITY_ENRICHMENT_URL + "/LogBook/GetLogBook", headers=headers, data="logBookHeaderID="+logbook_header_id)
+
+    emptyCounter = 0
+    dateList = {}
+    date = 0
+    for days in json.loads(get_logBook_response.text)["data"]:
+        if days["id"] == "00000000-0000-0000-0000-000000000000":
+            dateList[date] = days["date"]
+            emptyCounter = emptyCounter + 1
+        date = date + 1
+
+    print(dateList)
+    if emptyCounter != 0:
+        if yn_prompt("There are currently " + str(emptyCounter) + " empty slot(s) in your logBook! Do you want to set it to OFF?"):
```

Comment:
The explicit cast to string is unnecessary here, and you can get the number from empty_dates length
```suggestion
        if yn_prompt("There are currently " + len(empty_dates) + " empty slot(s) in your logBook! Do you want to set it to OFF?"):
```